src/routes.jsx

```diff
@@ -81,7 +81,7 @@ export const makeRoutes = (routing) => {
                 <Route path="/newstudy" onEnter={()=>{$(document).scrollTop(0)}} getComponent={lazyLoadComponent(StudyViewPage)} />
                 <Route path="/study" component={getBlankPage()} />
 
-                <Route path="/mutation_mapper" getComponent={lazyLoadComponent(MutationMapperTool)} />
+                <Route path="/s/mutation_mapper" getComponent={lazyLoadComponent(MutationMapperTool)} />
```

Comment:
Do we still need to keep the original path `/mutation_mapper`? I think the only place where we refer this route is in `mutation_mapper.jsp` which we can retire now.